doc/api/dns.md

```diff
@@ -387,7 +387,7 @@ changes:
 
 * `hostname` {string} Host name to resolve.
 * `options` {Object}
-  * `ttl` {boolean} Retrieve the Time-To-Live value (TTL) of each record.
+  * `ttl` {boolean} Retrieves the Time-To-Live value (TTL) of each record.
```

Comment:
Not sure this is correct, the option doesn't retrieve anything, however setting it to true instruct the API to retrieve the value.

Reply:
yes, I understand, but it does retrieve the time to live value when ttl option is used, in this case the ttl is 60